src/main/java/gov/nasa/cumulus/metadata/aggregator/UMMGranuleFile.java

```diff
@@ -515,6 +406,132 @@ private JSONObject exportSpatial() {
                 }
             }
         }
+
+        // We can only include orbital or bounding-box data, not both
+        if (foundOrbitalData == false) {
+
+            horizontalSpatialDomain.put("Geometry", geometry);
+
+            JSONArray boundingRectangles = new JSONArray();
+            geometry.put("BoundingRectangles", boundingRectangles);
+
+            double north = 0, south = 0, east = 0, west = 0;
+            if(granule !=null && granule instanceof  gov.nasa.cumulus.metadata.aggregator.UMMGranule) {
+                east = ((UMMGranule) granule).getBbxEasternLongitude() != null ?
+                        ((UMMGranule) granule).getBbxEasternLongitude() : 0;
+                west = ((UMMGranule) granule).getBbxWesternLongitude() != null?
+                        ((UMMGranule) granule).getBbxWesternLongitude() : 0;
+                north = ((UMMGranule) granule).getBbxNorthernLatitude() != null?
+                        ((UMMGranule) granule).getBbxNorthernLatitude() : 0;
+                south = ((UMMGranule) granule).getBbxSouthernLatitude() != null?
+                        ((UMMGranule) granule).getBbxSouthernLatitude() : 0;
```

Comment:
This section is new--looks like it's just making sure there are north/south/east/west values present? Setting to 0 if they're null?

Reply:
Right, if I remember correctly, this was related to some granules we encountered with bad spatial metadata, but the UMM-C still required spatial extent, but there were also just a lot of little differences between the internal codebase we were working from, and the previous GitHub repository.